### PR TITLE
Close all issues: 8.1.0, more tests, python 3.10, pypy37, maitainer update

### PR DESCRIPTION
**Pull request by hmaarrfk**
- Closes #44
- Closes #43
- Closes #42
- Closes #40
- Closes #35
- Closes #46 

Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by ngam** [comment]
@hmaarrfk let's run the whole test suite please

**Comment by ngam** [comment]
`make` and then `make test`

**Comment by ngam** [comment]
https://github.com/PyAV-Org/PyAV/blob/main/HACKING.rst

**Comment by hmaarrfk** [comment]
> make and then make test

Thanks for the suggestion!

**Comment by hmaarrfk** [comment]
@conda-forge-admin please rerender

**Comment by ngam** [comment]
Thanks @hmaarrfk this looks good!

**Comment by xhochy** [comment]
@hmaarrfk Should we merge this?

**Comment by hmaarrfk** [comment]
I think so!

**Comment by hmaarrfk** [comment]
@conda-forge-admin please rerender

**Comment by hmaarrfk** [comment]
I can't merge this.

**Comment by hmaarrfk** [comment]
@conda-forge-admin please rerender

**Comment by hmaarrfk** [comment]
@xhochy does travis no longer exist? This was having trouble emulating for ppc64le

**Comment by hmaarrfk** [comment]
https://app.travis-ci.com/github/conda-forge/av-feedstock/pull_requests hmm, i don't see the build...

**Comment by h-vetinari** [comment]
Need to run emulation on azure for ppc. Travis is dead (or as near as makes no difference).

**Comment by hmaarrfk** [comment]
How do you run emulation? I thought it was enabled by default. I think an audio test just failed. Likely bad emulation.

File: conda-forge.yml [hunk 1]
```diff
@@ -1,4 +1,8 @@
-build_platform: {osx_arm64: osx_64}
+build_platform:
+  osx_arm64: osx_64
 conda_forge_output_validation: true
-provider: {win: azure}
+provider:
+  linux_aarch64: default
+  linux_ppc64le: default
```
Comment (h-vetinari) [inline conda-forge.yml:6]: > How do you run emulation? 
```suggestion
  linux_ppc64le: azure
```
plus rerender

Comment (hmaarrfk) [reply conda-forge.yml:6]: isn't default = azure?

**Comment by hmaarrfk** [comment]
@conda-forge/admin please rerender

**Comment by hmaarrfk** [comment]
@conda-forge-admin, please rerender

File: conda-forge.yml (hunk 1 above)
Comment (hmaarrfk) [reply conda-forge.yml]: Azure is default. My thinking is that a nice test failed.

Comment (h-vetinari) [reply conda-forge.yml]: OK, that's a relatively recent change. I had only responded to your "how to run in emulation", sorry.

Comment (h-vetinari) [reply conda-forge.yml]: In any case, CI is green now 🥳

Comment (hmaarrfk) [reply conda-forge.yml:6]: Gotcha! Thanks for chiming in.

Yeah, I can't merge here. I force pushed to remove the last commit.

**Comment by hmaarrfk** [comment]
Thakn you! Lets rebuild once ffmpeg is fixed.

**Pull request merged**